src/couch_mrview_http.erl

```diff
@@ -576,6 +576,9 @@ check_view_etag(Sig, Acc0, Req) ->
 
 
 parse_json(V) when is_list(V) ->
-    ?JSON_DECODE(V);
+    case lists:all(fun is_binary/1, V) of
```

Comment:
the original cloudant multi-query solved it by cut'n'pasting couch_mrview_http:parse_params into chttpd_view.erl and removing json decode from the keys in question. to me it looks like more of a hack, but do you believe this is better approach?